src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/properties.cpp

```diff
@@ -89,11 +89,12 @@ INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory, OVCheckChangePropComplieModleGetPr
         OVCheckChangePropComplieModleGetPropTests_DEVICE_ID::getTestCaseName);
 
 /* Add prefix mandatory_ to suffix (getTestCaseName) of HW plugin test cases */
-INSTANTIATE_TEST_SUITE_P(
-    ov_plugin,
-    OVCheckChangePropComplieModleGetPropTests_InferencePrecision,
-    ::testing::Combine(::testing::Values(ov::test::utils::target_device), ::testing::Values(ov::AnyMap({}))),
-    MARK_MANDATORY_FOR_HW_DEVICE(OVCheckChangePropComplieModleGetPropTests_InferencePrecision::getTestCaseName));
+INSTANTIATE_TEST_SUITE_P(ov_plugin,
+                         OVCheckChangePropComplieModleGetPropTests_InferencePrecision,
+                         ::testing::Combine(::testing::Values(ov::test::utils::target_device),
+                                            ::testing::Values(ov::AnyMap({}))),
+                         MARK_MANDATORY_PROPERTY_FOR_HW_DEVICE(
+                             OVCheckChangePropComplieModleGetPropTests_InferencePrecision::getTestCaseName));
```

Comment:
This changed is covered by https://github.com/openvinotoolkit/openvino/pull/26250